wasm/jit/jit_amd64_test.go

```diff
@@ -1404,6 +1402,58 @@ func TestAmd64Builder_handleCall(t *testing.T) {
 	})
 }
 
+func TestAmd64Builder_handleMemoryGrow(t *testing.T) {
+	builder := requireNewBuilder(t)
+
+	builder.initializeReservedRegisters()
+	// Emit memory.grow instructions.
+	builder.handleMemoryGrow()
+
+	// Compile.
+	code, err := builder.assemble()
+	require.NoError(t, err)
+
+	// Run code.
+	eng := newEngine()
+	mem := newMemoryInst()
+	jitcall(
+		uintptr(unsafe.Pointer(&code[0])),
+		uintptr(unsafe.Pointer(eng)),
+		uintptr(unsafe.Pointer(&mem.Buffer[0])),
+	)
+	// The function call must end with jitCallStatusCodeCallBuiltInFunction status.
```

Comment:
change this to answer why or remove it as the assertion already says what

Reply:
deleted https://github.com/tetratelabs/wazero/pull/85/commits/8389a7f67a6fd22b32c71996940f2108d7373f46